extensions/historical-imagery/tests/historical-imagery_collection.test.mjs

```diff
@@ -98,7 +99,7 @@ o.spec('Historical Imagery Extension Collection', () => {
     // given
     for (const role of ['producer', 'licensor', 'processor', 'host']) {
       const example = JSON.parse(await fs.readFile(examplePath));
-      example.providers = example.providers.filter((provider) => !role in provider.roles);
```

Comment:
I know you didn't change this, However I am not entirely sure what this is trying to achieve ? (and if it actually works?) `in` is not really used in javascript so kinda odd to see it here?

Reply:
Fixed in #280.